docs/deployment/deployment.md

```diff
@@ -71,6 +71,9 @@ ENABLE_CUSTOMER_USAGE_ATTRIBUTION <br>CUSTOMER_USAGE_ATTRIBUTION_ID | No | By de
 ENABLE_DEV_CODE | No | Defaults to `false`. It is not recommended to enable this flag, it is for development testing scenarios only.
 APPLICATION_TITLE | No | Defaults to "". Providing a value for this parameter will replace the Information Assistant's title in the black banner at the top of the UX.
 ENABLE_BING_SAFE_SEARCH | No | Defaults to `true` If you are using the Bing Search API, you can set the following values to enable safe search.
+SHAREPOINT_SITES | No | Delimited comma list of sharepoint sites you wish you ingest, this shares a 1:1 relationship with **SHAREPOINT_FOLDERS** Information on setting up Sharepoint Ingestion can be found here [Sharepoint Feature](/docs/features/sharepoint.md)
```

Comment:
I would combine this and the Folders into one cell in the table and describe the relationship between them better. See the `local.env.example` updates in the other PR.